jarbas/dashboard/admin.py

```diff
@@ -311,6 +325,12 @@ def suspicious(self, obj):
     suspicious.short_description = 'suspeito'
     suspicious.boolean = True
 
+    def receipt(self, obj):
+        return obj.receipt_fetched is not False
```

Comment:
This is not the best way to do it. Jarbas could have tried to fetch the receipt (`receipt_fetched = True`) but still the URL wasn't available (`receipt_url = None`). I would put `obj.receipt_url is not False` or `bool(obj.receipt_url)`…